chromium_src/components/permissions/request_type.cc

```diff
@@ -5,19 +5,65 @@
 
 #include "components/permissions/request_type.h"
 
-#define BRAVE_GET_ICON_ID_DESKTOP \
-  case RequestType::kWidevine:    \
+#include "brave/components/brave_wallet/common/buildflags/buildflags.h"
+#include "third_party/widevine/cdm/buildflags.h"
+
+// TODO(jocelyn): Might need to update icon when we have ethereum.enable UI
+// support in Android.
+#define BRAVE_GET_ICON_ID_ANDROID   \
+  case RequestType::kBraveEthereum: \
+    return IDR_ANDROID_INFOBAR_PERMISSION_COOKIE;
```

Comment:
cookie?

Reply:
we haven't finished android support of ethereum.enable, so i'm just using a existing value first to avoid build break first, and we could change this while doing android support